### PR TITLE
Includes description of required headers

### PR DESCRIPTION
**Pull request by dafnalia**

### CODE REVIEW

File: apix_documentation_swagger_v3.yaml [hunk 1]
```diff
@@ -147,6 +147,24 @@ info:
...
+        * __Date__ It requires the same format used in the generation of the
+          checksum.
+
+          `-H 'Date: Tue, 24 Apr 2018 21:04:28 GMT'`
+        * __Content-Type__ Always should be application/json
+
+          `-H 'Content-Type: application/json'`
+        * __Accept__ Always should be application/vnd.regalii.v3.2+json
+
+          `-H 'Accept: application/vnd.regalii.v3.2+json'`
```
Comment (romariolopezc) [inline apix_documentation_swagger_v3.yaml]: I think the version (3.2) can be taken from a variable and use it to generate the docs. This way doesn't need to be hardcoded and when we change the version, this change automatically

Comment (dafnalia) [reply apix_documentation_swagger_v3.yaml]: Is it ok with you if I create a ticket for that?

Comment (romariolopezc) [reply apix_documentation_swagger_v3.yaml]: Yes, we should create a ticket for everything regarding the documentation

Comment (dafnalia) [reply apix_documentation_swagger_v3.yaml]: Jajaja yes, I meant if you agreed on creating a ticket vs figuring out how to do it right now

Comment (dafnalia) [reply apix_documentation_swagger_v3.yaml]: https://arcusfi.atlassian.net/browse/USX-353

**Pull request merged**